website/common/locales/en/emptyArmory.json

```diff
@@ -0,0 +1,22 @@
+{
+    "emptyArmoryNew": "Empty the armory: New Challenge Available!",
```

Comment:
"Empty the Armory" should be written with capital E and A everywhere that it occurs, since it's the title of the feature.